src/wdio/services/TerraCommands/visual-regression.js

```diff
@@ -54,8 +16,14 @@ const themeEachCustomProperty = (...args) => {
 
   Object.entries(styleProperties).forEach(([key, value]) => {
     global.it(`themed [${key}]`, () => {
+      // add document level style override
       global.browser.execute(`document.documentElement.style.setProperty('${key}', '${value}')`);
+
+      // take screenshot
       global.expect(global.browser.checkElement(selector)).to.matchReference();
+
+      // remove documented defined style override
+      global.browser.execute(`document.documentElement.style.setProperty('${key}', '')`);
```

Comment:
Do we want to remove the style or set it to the previously defined style? I'm thinking of running a theme test against something like the clinical theme. That might not be a valid use case, maybe we say we only run theme tests against the default styles.

Reply:
As of now we only run the default theme. I thought this was weird too, but I think its because we are applying the themed css-var value at a higher level than where the css-var is being pulled in on the page. I tested this on terra-ui with color and font size and this is what I was seeing:

![Screen Shot 2019-05-15 at 2 06 40 PM](https://user-images.githubusercontent.com/14099737/57802025-bcafbe80-771a-11e9-8a0b-b7ab49c30b96.png)

![Screen Shot 2019-05-15 at 2 04 36 PM](https://user-images.githubusercontent.com/14099737/57801963-9722b500-771a-11e9-8be8-7c56317cdad2.png)